src/components/v2/V2Project/V2ProjectReconfigureModal/V2ReconfigureModalTrigger.tsx

```diff
@@ -0,0 +1,77 @@
+import { Button, Tooltip } from 'antd'
+import { V1ProjectContext } from 'contexts/v1/projectContext'
+import React, { useContext, useRef, useState } from 'react'
+import { Provider } from 'react-redux'
+import store, { createStore } from 'redux/store'
+import { Trans } from '@lingui/macro'
+
+import { BigNumber } from '@ethersproject/bignumber'
+
+import { SettingOutlined } from '@ant-design/icons'
+
+import V2ProjectReconfigureModal from './index'
+
+// This component uses a local version of the entire Redux store
+// so editing within the Reconfigure Funding modal does not
+// conflict with existing Redux state. This is so editing a
+// persisted Redux state and the Reconfigure Funding modal
+// are independent.
+export default function V2ReconfigureFundingModalTrigger({
+  fundingDuration,
+}: {
+  fundingDuration?: BigNumber
+}) {
+  const { isPreviewMode } = useContext(V1ProjectContext)
+
+  const localStoreRef = useRef<typeof store>()
+
+  const [reconfigureModalVisible, setReconfigureModalVisible] =
+    useState<boolean>(false)
+
+  function handleModalOpen() {
+    localStoreRef.current = createStore()
+    setReconfigureModalVisible(true)
+  }
+
+  return (
+    <div style={{ textAlign: 'right' }}>
+      {/* {fundingDuration?.gt(0) ? (
```

Comment:
@johnnyd-eth in general lets try to minimize commented code in the codebase. Find to leave this one though